package.json

```diff
@@ -6,20 +6,22 @@
   "main": "dist/react-toggled.cjs.js",
   "jsnext:main": "dist/react-toggled.esm.js",
   "module": "dist/react-toggled.esm.js",
+  "typings": "typings/index.d.ts",
```

Comment:
We'll want to add `typings` to the `files` property so this is published to npm with everything else.